app/controllers/Application.scala

```diff
@@ -45,6 +46,11 @@ class Application(
     Redirect(location + path, request.queryString)
   }
 
+  def unsupportedBrowser: Action[AnyContent] = CachedAction() { implicit request =>
+    BrowserCheck.logUserAgent
```

Comment:
I would write `BrowserCheck.logUserAgent(request)`

see https://stackoverflow.com/questions/7600910/difference-between-function-with-parentheses-and-without

Reply:
Ok, can do.